libsnark/gadgetlib1/gadgets/fields/fp2_gadgets.hpp

```diff
@@ -50,12 +50,16 @@ class Fp2_variable : public gadget<typename Fp2T::my_Fp> {
 
     void generate_r1cs_equals_const_constraints(const Fp2T &el);
     void generate_r1cs_witness(const Fp2T &el);
-    Fp2T get_element();
+    Fp2T get_element() const;
 
     Fp2_variable<Fp2T> operator*(const FieldT &coeff) const;
+    Fp2_variable<Fp2T> operator*(const Fp2T &fp2_const) const;
     Fp2_variable<Fp2T> operator+(const Fp2_variable<Fp2T> &other) const;
     Fp2_variable<Fp2T> operator+(const Fp2T &other) const;
```

Comment:
What about renaming `other` as `fp2_const` here as per your comment https://github.com/clearmatics/libsnark/pull/10/files#r449134158 ?

Reply:
renamed

Reply:
Did you push your changes @dtebbs ?

Reply:
Ok, checked https://www.githubstatus.com/ which indicates a degraded service.. that may be the reason why I can't see your changes

Reply:
I did push 1 new commit 0d2cb1600632d2cad233a826bffc34eada909f9c, but it might need some time to propagate.  If it still doesn't appear on your side let me know and I can force push with a different commit id.

Reply:
All good, I think Github was running in degraded mode yesterday. I can see your commit now, thanks